### PR TITLE
Feature cart

### PR DESCRIPTION
**Pull request by Crysicia**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -31,4 +31,8 @@ Metrics/BlockLength:
     - 'vendor/bundle'
 
 Style/ClassAndModuleChildren:
-  Enabled: false
\ No newline at end of file
+  Enabled: false
+  
+RSpec/InstanceVariable:
+  Exclude:
+    - 'spec/**/*.rb'
```
Comment (stephan-dev) [inline .rubocop.yml:38]: acab

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -142,7 +142,7 @@
   # initial account confirmation) to be applied. Requires additional unconfirmed_email
   # db field (see migrations). Until confirmed, new email is stored in
   # unconfirmed_email column, and copied to email column on successful confirmation.
-  config.reconfirmable = true
+  config.reconfirmable = false
```
Comment (stephan-dev) [inline config/initializers/devise.rb:145]: ok, me souviens pas avoir mis ça, si c'est moi c'était une erreur.

File: spec/factories/users.rb [hunk 3]
```diff
@@ -19,7 +19,7 @@
 
 FactoryBot.define do
   factory :user do
-    password = Faker::Internet.password
+    password = Faker::Internet.password(6, 12)
```
Comment (stephan-dev) [inline spec/factories/users.rb:22]: I had looked for that but thought with devise it was too complex to test, is it really possible?

File: spec/views/carts/edit.html.erb_spec.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "carts/edit.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (stephan-dev) [inline spec/views/carts/edit.html.erb_spec.rb:6]: ça ne bloque plus maintenant, les pending tests?

**Review by stephan-dev — APPROVED** [review]
good for me, see small comments.

File: .rubocop.yml (hunk 1 above)
Comment (Crysicia) [reply .rubocop.yml:38]: No, I hate Moby Dick

File: config/initializers/devise.rb (hunk 2 above)
Comment (Crysicia) [reply config/initializers/devise.rb:145]: It was enabled by default :)

File: spec/factories/users.rb (hunk 3 above)
Comment (Crysicia) [reply spec/factories/users.rb:22]: Yup

File: spec/views/carts/edit.html.erb_spec.rb (hunk 4 above)
Comment (Crysicia) [reply spec/views/carts/edit.html.erb_spec.rb:6]: It never did

**Pull request merged**